lib/ansible/modules/cloud/amazon/rds_param_group.py

```diff
@@ -48,31 +48,47 @@
         - 'aurora5.6'
         - 'mariadb10.0'
         - 'mariadb10.1'
+        - 'mariadb10.2'
```

Comment:
What do you think about pointing to AWS/[boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds.html#RDS.Client.create_db_parameter_group) docs? Otherwise, this looks like busywork to always be maintaining. The boto3 docs don't even have a comprehensive list, just direct people to check with the CLI: `aws rds describe-db-engine-versions --query "DBEngineVersions[].DBParameterGroupFamily"`